src/client/pages/Varen/Varen.tsx

```diff
@@ -0,0 +1,170 @@
+import { useMemo } from 'react';
+
+import styles from '../Vergunningen/Vergunningen.module.scss';
+import { AppRoutes } from '../../../universal/config/routes';
+import { isError, isLoading } from '../../../universal/helpers/api';
+import { defaultDateFormat } from '../../../universal/helpers/date';
+import { getCustomTitleForVergunningWithLicensePlates } from '../../../universal/helpers/vergunningen';
+import { CaseType } from '../../../universal/types/vergunningen';
+import {
+  ErrorAlert,
+  Linkd,
+  MaintenanceNotifications,
+  PageContent,
+  PageHeading,
+  SectionCollapsible,
+  Table,
+  ThemaIcon,
+  addTitleLinkComponent,
+} from '../../components';
+import { OverviewPage } from '../../components/Page/Page';
+import { ThemaTitles } from '../../config/thema';
+import { useAppStateGetter } from '../../hooks/useAppState';
+import { hasMultiplePermits } from '../VergunningDetail/WVOS';
+import { Vergunning } from '../../../server/services';
+
+export const DISPLAY_PROPS = {
+  identifier: 'Kenmerk',
+  title: 'Soort vaar',
+  dateRequest: 'Aangevraagd',
+};
+
+export const DISPLAY_PROPS_HISTORY = {
+  identifier: 'Kenmerk',
+  title: 'Soort vaar',
+  decision: 'Resultaat',
+};
+
+const titleTransformMap: Record<string, any> = {
+  [CaseType.TouringcarJaarontheffing]:
+    getCustomTitleForVergunningWithLicensePlates,
+  [CaseType.TouringcarDagontheffing]:
+    getCustomTitleForVergunningWithLicensePlates,
+  [CaseType.EigenParkeerplaats]: getCustomTitleForVergunningWithLicensePlates,
+  [CaseType.EigenParkeerplaatsOpheffen]:
+    getCustomTitleForVergunningWithLicensePlates,
+};
+
+export default function Varen() {
+  const { VAREN } = useAppStateGetter();
+
+  const Varen: Vergunning[] = useMemo(() => {
+    if (!VAREN.content?.length) {
+      return [];
+    }
+    const items: Vergunning[] = VAREN.content
+      .filter((x) => x)
+      .map((item) => {
+        return {
+          ...item,
+          title:
+            item.caseType in titleTransformMap
+              ? titleTransformMap[item.caseType](item)
+              : item.title,
+          dateRequest: defaultDateFormat(item.dateRequest),
+        };
+      });
+    return addTitleLinkComponent(items, 'identifier');
+  }, [VAREN.content]);
```

Comment:
Deze functionalteit mak allemaal in de `useVarenThemaData` hook.

```suggestion
  const {... } = useVarenThemaData();
```